### ISSUE TITLE
Add Overhead Panel

### ISSUE THREAD

**Issue by HHS81** [HHS81/do328#1]
add textured 3-model of the Overhead panel

**Comment by xcvb85** [comment]
This is what I miss the most (without overhead panel no better systems and without better systems no better EICAS).

**Comment by HHS81** [comment]
Inkscape is quite slow here on my system, hope to get it ready today. I'm just fixing the exterior lighting

**Comment by xcvb85** [comment]
Today? Wow, this sounds cool! :grin:

**Comment by HHS81** [comment]
Just the overhead panel. The whole flightdeck needs much more work. :smile:

**Comment by HHS81** [comment]
Didn't make it today:
Inkscape is getting really slow with lots of text:
![ovh](https://cloud.githubusercontent.com/assets/12649375/20652413/f9565a42-b4f8-11e6-82fa-d6d9f02e6a26.jpg)

**Comment by xcvb85** [comment]
Wow, this looks nice! :+1: 
Can't wait to see it on the plane.

**Comment by xcvb85** [comment]
Any chance to see this on the plane soon?

**Comment by xcvb85** [comment]
OK, the OHP has its own issue. This is my current version:

![ovh](http://i65.tinypic.com/161fc4y.png)

Not sure if I missed anything. The black bar is the placeholder for the button icons. I hope the space is sufficient. The 7-segment textures are not the correct ones yet - also just placeholders.

**Comment by HHS81** [comment]
Wow- great work! The fuse/ breaker will be much better readible! 👍

**Comment by xcvb85** [comment]
I wonder how to add illumination. Emissive materials, light maps or something else? I need it for the LEDs, 7-segment displays and button icons.

![ovh](http://i63.tinypic.com/2lkvigj.jpg)

**Comment by HHS81** [comment]
The panel is backlit (Text and lines),  so I prefer a lightmap. The Button Icons,  LEDs and 7segment Displays can be made by Material animation.

**Comment by xcvb85** [comment]
OK, I will try. The button icons are as least as annoying as the circuit breaker texts. This will take some time so please be patient.

**Comment by HHS81** [comment]
You will need only one text for one sort of icon. Not for every Button. 
German: die Icons können per select  animation animiert werden.  Leuchten können Sie entweder per material animation  (emission) oder wenn sie auf der gleichen Textur sind wie das Panel dann per lightmap. Da die Icons wohl kaum variieren,  reichen für pro Variation ein Text.  

Ich hoffe Du hast den Text als eigenes Layer / Ebene gespeichert?

**Comment by xcvb85** [comment]
Leider ist das nicht einfach nur an/aus, sondern z.B. bei Non-Essential Bus entweder der Balken, oder "OFF". Ich muss noch überlegen, wie man das am Besten macht. Ich schaue mir das mal bei der 777 oder dem A330 an. Ich denke entweder zwei Icons oder Masken verwenden.

**Comment by HHS81** [comment]
Ja,  ist für mich auch komplettes Neuland.

**Comment by ghost** [comment]
Für unterschiedliche Texturen benutze ich immer textranslate,ist glaub ich das Sinnvollste

**Comment by xcvb85** [comment]
Ja, entweder zwei Icons mit Textranslate wechseln, oder eine Maske mit Select oder Translate. Mal schauen, wie viele Icons es am Ende werden. Das Blöde an textranslate ist, dass es keine Expressions unterstützt. Ich bin noch am Überlegen, wie ich stattdessen die 7-Segment Anzeigen umsetzte, denn dafür bräuchte ich Modulo.

**Comment by HHS81** [comment]
OSG-Text wäre ne Möglichkeit,  den Font dazu haben wir ja seit einiger Zeit. Oder klassisch per texttranslate wie bisher

**Comment by xcvb85** [comment]
Für die 7-Segment Anzeigen müsste ich bei textranslate eine eigene Textur erstellen, damit es nach Überschreiten der 9 wieder von vorne beginnt (normalerweise macht man sowas mit Modulo, aber das geht ja leider nicht). Canvas könnte an auch nehen, wobei der Font nicht ganz dem Original entspricht. Die Spannungensanzeigen von DC Bus 1 & 2 sind schräg mit abgeschrägten Kanten und die Kabinendruckanzeige ist gerade ohne abgeschrägte Kanten.

**Comment by HHS81** [comment]
Es gab ne Zeit vor Canvas und OSG-TEXT und da hatte man es in der Tat mit einer eigenen Textur gemacht. Erlaubte gewisse grafische Effekte,  die so sonst nicht so einfach möglich wären.
